vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -0,0 +1,175 @@
+---
+sidebarDepth: 2
+---
+# Manage NGINX
```

Comment:
I think we need an overview section to introduce the page. Our starting point assumes someone has already installed the NGINX controller and this page is for verifying and refining that configuration.